viewpump/build.gradle

```diff
@@ -0,0 +1,31 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 24
+    buildToolsVersion "24.0.0"
+
+    defaultConfig {
+        minSdkVersion 7
```

Comment:
I think I mentioned in the other review, that we should probably support 14+ which will simplify the `Inflater` a little bit.